cmd/pulumi-language-yaml/testdata/projects/l1-output-string/Main.yaml

```diff
@@ -1,9 +1,9 @@
 outputs:
```

Comment:
Accidentally commit the broken version of this file in https://github.com/pulumi/pulumi-yaml/pull/673